lib/rubocop/cop/mixin/safe_assignment.rb

```diff
@@ -14,7 +14,7 @@ module SafeAssignment
       def_node_matcher :empty_condition?, '(begin)'
 
       # @!method setter_method?(node)
-      def_node_matcher :setter_method?, '[(send ...) setter_method?]'
+      def_node_matcher :setter_method?, '[({send csend} ...) setter_method?]'
```

Comment:
Can you use `call` instead?

```suggestion
      def_node_matcher :setter_method?, '[(call ...) setter_method?]'
```